public/index.html

```diff
@@ -155,7 +155,45 @@ <h3 class="acc-header">
+          <path d="M59.7502 54.3135V38.3755H56.8984V54.3135H59.7502ZM64.4833 54.3135V43.1524H64.5229L68.0083 54.3135H71.2562V38.3755H68.6421V49.291H68.6025L65.1566 38.3755H61.8692V54.3135H64.4833ZM78.4251 54.3135V41.0095H81.5145V38.3755H72.484V41.0095H75.5734V54.3135H78.4251ZM90.8421 54.3135V51.6795H85.5941V47.3936H90.347V44.7596H85.5941V41.0095H90.644V38.3755H82.7423V54.3135H90.8421ZM95.1197 54.3135V47.5722H96.6446C97.2387 47.5722 97.6612 47.7508 97.912 48.1079C98.1629 48.4651 98.3015 48.9562 98.3279 49.5812L98.4071 52.6393C98.4203 52.9518 98.4467 53.2532 98.4863 53.5434C98.5259 53.8335 98.6183 54.0903 98.7636 54.3135H101.853V54.1795C101.589 54.0158 101.417 53.7108 101.338 53.2643C101.285 52.8179 101.246 52.1706 101.219 51.3223C101.206 50.8908 101.193 50.5001 101.18 50.1504C101.166 49.8007 101.147 49.477 101.12 49.1794C101.041 48.2865 100.836 47.6392 100.506 47.2374C100.176 46.8356 99.6547 46.5751 98.9418 46.4561V46.4114C99.7472 46.218 100.341 45.7827 100.724 45.1056C101.107 44.4285 101.298 43.5542 101.298 42.4828C101.298 41.0988 100.968 40.0682 100.308 39.3911C99.6481 38.714 98.724 38.3755 97.5357 38.3755H92.2679V54.3135H95.1197ZM96.3079 45.34H95.1197V40.7416H96.4663C97.7866 40.7416 98.4467 41.4783 98.4467 42.9515C98.4467 43.8146 98.2619 44.4285 97.8922 44.7931C97.5225 45.1577 96.9944 45.34 96.3079 45.34ZM105.893 54.3135V43.1524H105.933L109.418 54.3135H112.666V38.3755H110.052V49.291H110.012L106.566 38.3755H103.279V54.3135H105.893ZM122.746 54.3135V51.6795H117.498V47.3936H122.251V44.7596H117.498V41.0095H122.548V38.3755H114.646V54.3135H122.746ZM129.36 54.3135V41.0095H132.45V38.3755H123.419V41.0095H126.509V54.3135H129.36Z" fill="#164176"/>
+        </svg>
+      </a>  
+    </div>
+    <div class="footer-banner">
+      <div class="footer-links">
+        <a class="footer-link" href="https://github.com/morehumaninternet/frontend" aria-label="Github"><svg width="25" height="25" viewBox="0 0 31 30" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
All the links work great. Can you please open them in a new tab?

Reply:
added new tabs for all footer's links.